setup.py

```diff
@@ -61,7 +61,7 @@
     "sphinx-multiversion",
     "timeout-decorator",
     "torch>=1.10,!=1.12.0",
-    "transformers==4.33.3",
+    "transformers==4.35.0",
```

Comment:
Can we update to the latest HF release, which is v4.35.2?

Reply:
I updated it to v4.35.2 now :+1: